tests/triggers/test_text_triggers.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+from ...gramhopper.triggers.text_triggers import _RegExpTrigger, \
+    _HasSubstringTrigger, _HasExactWordTrigger
+
+
+@pytest.mark.usefixtures('update')
+class TestRegexpTrigger:
+    def test_regexp_trigger(self, update):
+        email_pattern = r'^([a-z0-9_\.-]+)@([\da-z\.-]+)\.([a-z\.]{2,6})$'
```

Comment:
Why is this the regex signature of an email address? Can't the domain contain digits? (e.g. nana10) 
That's not crucial since we're trying to test the regex trigger, but it bugs me ¯\_(ツ)_/¯

Reply:
I fully agree with the last sentence. I'll try to find an alternative regexp.

Reply:
I've just noticed it has `\d` in the domain part.
Anyway, I obviously didn't write it by myself - I've copied it from somewhere.

Also, you've made me notice that the last match group isn't necessarily 2-6 characters (for example, domains that end with ".academy").

Reply:
Oh nice